tests/write.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
```

Comment:
If this can be switched to `/bin/sh`, that would help. Periodically I verify fwup on BSDs and remove all of the bash-isms.

Also, could you add comments to the top of the shell scripts describing their purpose.